phonecall_plan/README.rst

```diff
@@ -0,0 +1,70 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+      :alt: License: AGPL-3
+
+==============
+phonecall_plan
+==============
+
+This module add a wizard to allow you to plan call from lead/partner view.
+
+Installation
```

Comment:
Remove the section which does not add any added value (this is the standard process)